pkg/pxc/app/statefulset/proxysql.go

```diff
@@ -108,6 +102,24 @@ func (c *Proxy) AppContainer(spec *api.PodSpec, secrets string, cr *api.PerconaX
 		SecurityContext: spec.ContainerSecurityContext,
 	}
 
+	if cr.CompareVersionWith("1.5.0") < 0 {
+		appc.Env = append(appc.Env, corev1.EnvVar{
+			Name: "MYSQL_ROOT_PASSWORD",
+			ValueFrom: &corev1.EnvVarSource{
+				SecretKeyRef: app.SecretKeySelector(secrets, "root"),
+			},
+		})
```

Comment:
upgrade consistency test become broken because this line add MYSQL_ROOT_PASSWORD var to the end of array

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/452/commits/59f55b53163b0d98ff1a0a21838cf56dfa778ceb

https://github.com/percona/percona-xtradb-cluster-operator/pull/452/commits/319f19d5c01565df826489485f9c0d21cb201b1b